collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -789,6 +789,7 @@ void setTaskListener(TaskListener tl) {
                         .setFields("id")
                         .execute().getId();
             } catch (UserRecoverableAuthIOException e) {
+                createAlertDialog(getString(R.string.auth_failed_drive));
```

Comment:
Ah, ok. So I think we should show a message in the case of that `GoogleAuthIOException` that says something like "GoogleAuthIOException: if you are running a local build, please review the project readme." I believe this will never happen in production. @dcbriccetti does that sound right?